app/controllers/organizations_controller.rb

```diff
@@ -0,0 +1,174 @@
+class OrganizationsController < ApplicationController
+  def new
+    authorize Organization, :new?
```

Comment:
Asking for my dbout: bhaiya, there is two or three type of authorization, if we switch one to before_action, it will get executed in each call bhaiya, but that authorization won't be valid for all. Is there any way I can trigger different part of those three code as our requirements?